src/libraries/System.Formats.Tar/tests/TarFile/TarFile.ExtractToDirectoryAsync.Stream.Tests.cs

```diff
@@ -455,8 +462,13 @@ private Task ExtractToDirectory_ExactRootDirMatch_Links_Throws_Async(TarEntryFor
             string linkTargetFileName = "file.txt";
             string linkFileName = "link";
 
+#if TARGET_MACCATALYST || TARGET_IOS || TARGET_TVOS
+            // the folder used to store files needs to have a shorter path on Apple mobile platforms,
+            // because the TempDirectory gets created in folder with a path longer than 100 bytes
+            using TempDirectory root = new TempDirectory("/Users/helix-runner/Library/Developer/CoreSimulator/Devices/tempDir");
```

Comment:
Can you please store the string `"/Users/helix-runner/Library/Developer/CoreSimulator/Devices/tempDir"` in a field, to avoid repetition?